sample/build.gradle

```
@@ -0,0 +1,23 @@
+plugins {
+    id 'org.springframework.boot' version '2.7.9'
+    id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+    id 'java'
+}
+
+group = 'com.ibm.cloud'
```

Since we aren't going to publish this to maven central as an artifact itself I wonder if it is worth making this coordinate more example oriented
```suggestion
group = 'com.example'
```